src/webhooks/pubsub/index.ts

```diff
@@ -47,27 +45,27 @@ export const pubSubHandler = async (
     Buffer.from(request.body.message.data, 'base64').toString().trim()
   );
 
-  let repos,
-    octokit,
+  let octokit,
     now,
     code = 204;
   let func = new Map([
     ['stale-triage-notifier', notifyProductOwnersForUntriagedIssues],
     ['stale-bot', triggerStaleBot],
   ]).get(payload.name);
 
-  if (func === undefined || !payload.repos || !payload.repos.length) {
-    func = async () => {}; // no-op
-    code = 400;
-  } else {
-    repos = payload.repos;
+  // Validate dynamic method call as mentioned below
```

Comment:
Mentioned where below? At the link?

How about:

```suggestion
  // Performing the following check seems to suppress GitHub's dynamic method
  // call security warning.
```